addons/binding/org.openhab.binding.velux/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	* Copyright 2016-2018 (c) Guenther Schreiner <guenther.schreiner@smile.de>
+	*
+	* All rights reserved. This program and the accompanying materials
+	* are made available under the terms of the Eclipse Public License v1.0
+	* which accompanies this distribution, and is available at
+	* http://www.eclipse.org/legal/epl-v10.html
+	*
+-->
+<!-- Velux Bridge Things Binding -->
+<thing:thing-descriptions bindingId="velux" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Velux Bridge Binding -->
+	<bridge-type id="klf200">
+		<label>Velux KLF200 Bridge</label>
+		<description>The Velux KLF200 represents a gateway to all Velux devices.</description>
+
+		<channels>
+			<channel id="status" typeId="status" />
+			<channel id="firmware" typeId="firmware" />
```

Comment:
I'm sorry to stress you with this, but all bridge channels from firmware to products should not be channels. These information will not be part of any rule or smarthome related in any kind.
Please fill up thing/bridge properties with this information, so administrators can review them in Paper UI.

Reply:
The idea of having them as Channels is to show them on the GUI - not only during configuration phase.